Uzira.rte/Actors/Brains/Uzira/Armour.lua

```diff
@@ -0,0 +1,20 @@
+function Create(self)
+	local parent = self:GetParent();
```

Comment:
Shouldn't this adding armoured stuff be in OnAttach? I guess it's not a big deal, since there's no way to readd lost armour atm, but it seems more correct.

Reply:
Right, seeing as you've fixed the OnAttach bug, I'll do that